frontend/view-edit-client/client-section.component.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { useCss } from "kremling";
+
+export default function ClientSection(props: ClientSectionProps) {
```

Comment:
ClientSection is the collapsible card shown in the gif in the description. There will be one for Basic Info, Contact Info, Demographics, Intake Data